rust/src/openvasd/storage/mod.rs

```diff
@@ -383,10 +399,7 @@ async fn update_notus_feed(p: PathBuf, store: Arc<DefaultDispatcher>) -> Result<
                     filename: filename.to_owned(),
                 };
 
-                store.as_dispatcher().dispatch(
-                    &ContextKey::FileName(filename.to_owned()),
-                    Field::NotusAdvisory(Some(data).into()),
-                )?;
+                store.dispatch((), data)?;
```

Comment:
Doesn't that loose the filename information?

Reply:
The filename was never saved or used for Notus Advisories anyway. I also don't know if there is any use for them.